pkg/packagekit/autoupdates.jsx

```diff
@@ -90,6 +90,29 @@ class ImplBase {
         else
             this.supported = false;
     }
+
+    // Shared timer configuration logic
+    generateTimerConfigScript(configFile, day, time) {
+        let script = "";
+
+        if (time !== null || day !== null) {
+            if (day === "" && time === "") {
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-22515-acb2e3fc-20251126-085322-fedora-43-devel/Coverage/lcov/github-pr.diff.gcov.html)